modules/users/client/views/invite.client.view.html

```diff
@@ -27,6 +27,33 @@
           Codes are valid for 24 hours.
         </p>
 
+        <!-- Go to www.addthis.com/dashboard to customize your tools -->
+        <p>
+          <div ng-if="::invite.invitation.code">
+          <script type="text/javascript">
+            var code = document.getElementById("invite").value.toUpperCase();
+            var addthis_config = addthis_config||{};
+                addthis_config.ui_email_from = 'help@trustroots.org';
+                addthis_config.ui_email_note = 'Your invite code: ' + code + ' would be shared which can be used for signup with Trustroots at https://www.trustroots.org/signup?code=' + code;
```

Comment:
There's probably a way with `$location` (https://docs.angularjs.org/api/ng/service/$location) to make that domain dynamic, so that we can test this at dev.trustroots.org as well?